src/app/components/cds/buttons-checkboxes-chips/ButtonMain.module.css

```diff
@@ -311,11 +311,11 @@
     }
 
     &.outlined {
-      @mixin main-button-outlined-theme var(--color-gray-15), var(--color-gray-37), var(--color-gray-15), var(--color-gray-37), transparent;
+      @mixin main-button-outlined-theme var(--color-gray-15), var(--color-gray-37), var(--color-gray-15), var(--color-gray-37), var(--color-gray-96);
```

Comment:
Note to reviewers, this is unreleated to goal of this PR, but noticed this button main style issue